frontend/src/models/Exchange.model.ts

```diff
@@ -59,13 +59,17 @@ export const updateExchangeInfo = (federation: Federation): ExchangeInfo => {
 
 export interface Exchange {
   info: ExchangeInfo;
+  activeCoordinators: number;
   onlineCoordinators: number;
+  loadingCoordinators: number;
   totalCoordinators: number;
```

Comment:
activeCoordinators: # of coordinators marked as active by the user
  onlineCoordinators: # of online coordinators
  loadingCoordinators: # of coordinators with a pending request
  totalCoordinators: # of registered coordinators

Reply:
`activeCoordinators` maybe it is best to rename this one to `enabledCoordinators` for consistency and to not confuse these with the `online` coordinators. I believe we use `active` and `online` sometime as synonyms in the UI.